redfish/rules/rule.yaml

```diff
@@ -0,0 +1,85 @@
+Traverse:
```

Comment:
This file should become unnecessary in relation to `base-rules/dell-redfish-v117.yaml`.

Reply:
undo my changes

Reply:
I delete rule.yaml file